pkg/bgp/manager/manager_test.go

```diff
@@ -0,0 +1,173 @@
+// Copyright 2016-2021 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package manager
```

Comment:
There should be a newline separate the `package ...` line and the copyright header. The reason is because the godocs will pick the copyright header as the godoc for the package.